### ISSUE TITLE
Vector visualization lines don't un-draw

### ISSUE THREAD

**Issue by zellio** [jminuse/fluids#1]
If you are holding down right click and you drag your mouse off screen while the vector visualizations are drawing you will get a red line across the entire field of vision at approximately the angle of movement which doesn't un-draw.

**Comment by jminuse** [comment]
Good catch. Something is definitely up with the edges. The mass sources can
get stuck there as well. Thanks for checking it out, I'll try to figure it
out when I get home (my work computer can't run this simulation above 1
fps).
